phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/common/control/DetailsListElement.java

```diff
@@ -0,0 +1,109 @@
+package org.phoenicis.javafx.components.common.control;
+
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import org.phoenicis.javafx.components.common.skin.DetailsListElementSkin;
+import org.phoenicis.javafx.views.common.widgets.lists.AdditionalListWidgetInformation;
+import org.phoenicis.javafx.views.common.widgets.lists.ListWidgetEntry;
+import org.phoenicis.javafx.views.common.widgets.lists.details.DetailsListWidget;
+
+/**
+ * A list element component shown inside an {@link DetailsListWidget}
+ *
+ * @param <E> The concrete type of the element shown in this list element
+ */
+public class DetailsListElement<E> extends ControlBase<DetailsListElement<E>, DetailsListElementSkin<E>> {
+    /**
+     * The item represented by this list element
+     */
+    private final ObjectProperty<E> item;
+
+    /**
+     * The title belonging to the item
+     */
+    private final StringProperty title;
```

Comment:
Shouldn't this be "label"?

Reply:
I'm not sure, the variable was previously named `title` too.
It's also unrelated to it being shown as a label so I'm not sure.

Reply:
Ok.